bugbug/models/component.py

```diff
@@ -203,3 +204,70 @@ def is_meaningful(product, component):
 
     def get_feature_names(self):
         return self.extraction_pipeline.named_steps["union"].get_feature_names()
+
+    def check(self):
+        success = super().check()
+
+        # Check that the most meaningful product components stills have at
+        # least a bug in this component. If the check is failing that could
+        # means that:
+        # - A component has been renamed / removed
+        # - TODO: Complete this list
+
+        limit = 1
+
+        for product, component in self.meaningful_product_components:
+            query_data = {
+                # Search bugs in the given product and component
+                "product": product,
+                "component": component,
+            }
+
+            bugs_number = count_bugs(query_data)
+
+            if bugs_number != limit:
```

Comment:
Should be bugs_number == 0, right?

Reply:
Nice catch! I'm waiting the end of training process to run the check at least once before merging this PR